src/mono/System.Private.CoreLib/src/System/Threading/Thread.Mono.cs

```diff
@@ -206,15 +205,15 @@ private void Initialize()
         {
             InitInternal(this);
 #if TARGET_UNIX || TARGET_BROWSER
-            _waitInfo = new WaitSubsystem.ThreadWaitInfo(this);
+            EnsureWaitInfo();
```

Comment:
Can this be delete now that the waitInfo is always allocate lazily?

Reply:
Jupp, we could eliminate that now.